oneview/data_source_hypervisor_cluster_profile.go

```diff
@@ -0,0 +1,529 @@
+		for _, virtualSwitchPortUplink := range virtualSwitch.VirtualSwitchUplinks {
+			virtualSwitchPortUplinks = append(virtualSwitchPortUplinks, map[string]interface{}{
+				"action": virtualSwitchPortUplink.Action,
+				"active": virtualSwitchPortUplink.Active,
+				"mac":    virtualSwitchPortUplink.Mac,
+				"name":   virtualSwitchPortUplink.Name,
+				"vmnic":  virtualSwitchPortUplink.Vmnic,
+			})
+		}
+
+		/**********virtual switch upnlinks end************/
+
+		networkUris := make([]interface{}, len(virtualSwitch.NetworkUris))
+		for i, networkURI := range virtualSwitch.NetworkUris {
+			networkUris[i] = networkURI
+		}
+
+		virtualSwitches = append(virtualSwitches, map[string]interface{}{
+			"action":                     virtualSwitch.Action,
+			"name":                       virtualSwitch.Name,
+			"network_uris":               networkUris,
+			"version":                    virtualSwitch.Version,
+			"virtual_switch_port_groups": virtualSwitchPortGroups,
+			"virtual_switch_type":        virtualSwitch.VirtualSwitchType,
+			"virtual_switch_uplinks":     virtualSwitchPortUplinks,
+		})
+
+	}
+
+	/*****************virtual switch ends*************************************/
+
+	hypCPHHPTList := make([]map[string]interface{}, 0, 1)
+	hypCPHHPTList = append(hypCPHHPTList, map[string]interface{}{
+		"deployment_manager_type":      hypCP.HypervisorHostProfileTemplate.DeploymentManagerType,
+		"deployment_plan":              dplist,
+		"host_config_policy":           hostConfigPolicylist,
+		"host_prefix":                  hypCP.HypervisorHostProfileTemplate.Hostprefix,
+		"server_profile_template_uri":  hypCP.HypervisorHostProfileTemplate.ServerProfileTemplateUri.String(),
+		"virtual_switch_config_policy": virtualSwitchConfigPolicylist,
+		"virtual_switches":             virtualSwitches,
+	})
+
+	d.Set("hypervisor_host_profile_template", hypCPHHPTList)
+	d.Set("hypervisor_host_profile_uris", hypCP.HypervisorHostProfileUris)
+	d.Set("hypervisor_manager_uri", hypCP.HypervisorManagerUri)
+	d.Set("hypervisor_type", hypCP.HypervisorType)
+	ipPools := make([]interface{}, len(hypCP.IpPools))
+	for i, ipPool := range hypCP.IpPools {
+		ipPools[i] = ipPool
+	}
+	d.Set("ip_pools", ipPools)
+	d.Set("mgmt_ip_settings_override", hypCP.MgmtIpSettingsOverride)
+	d.Set("modified", hypCP.Modified)
+	d.Set("name", hypCP.Name)
+	d.Set("path", hypCP.Path)
+	d.Set("refresh_state", hypCP.RefreshState)
+	d.Set("scopes_uri", hypCP.ScopesUri)
+	sharedStorageVolumes := make([]interface{}, len(hypCP.SharedStorageVolumes))
+	for i, sharedStorageVolume := range hypCP.SharedStorageVolumes {
+		sharedStorageVolumes[i] = sharedStorageVolume
+	}
+	d.Set("shared_storage_volumes", sharedStorageVolumes)
+	d.Set("state", hypCP.State)
+	d.Set("state_reason", hypCP.StateReason)
+	d.Set("status", hypCP.Status)
+	d.Set("type", hypCP.Type)
+	d.Set("uri", hypCP.URI)
+	d.Set("soft_delete", true)
```

Comment:
Schema doesn't have soft_delete attribute. Remove this.

Reply:
done